test/client/debug.html

```diff
@@ -0,0 +1,104 @@
+<html>
```

Comment:
why did you add this file? We shouldn't have two files for client side tests (duplicated files, that is)

Reply:
this one serves unminified assets and makes debugging easier.